libraries/Microsoft.BotKit.Adapters.Slack/DialogData.cs

```diff
@@ -28,31 +28,37 @@ public DialogData(string title, string callback, string submit, List<DialogEleme
         /// <summary>
         /// Gets or Sets the tittle of the dialog.
         /// </summary>
+        /// <value>The tittle of the dialog.</value>
```

Comment:
Are the `<value>` necessary? Because, when compared with other libraries from BB-DotNet, the properties don't have any `<value>` tag (see [this](https://github.com/microsoft/botbuilder-dotnet/blob/master/libraries/Microsoft.Bot.Schema/Activity.cs#L252))

Reply:
Yes, I've noticed that, but for these files, there are warnings for the missing <value> tags.